src/platform/backends/libvirt/libvirt_virtual_machine.cpp

```diff
@@ -226,10 +226,13 @@ auto get_domain_definition(virConnectPtr connection, const mp::VirtualMachineDes
     return domain;
 }
 
-auto get_domain_state(virDomainPtr domain)
+auto domain_state_for(virDomainPtr domain)
```

Comment:
Can we move all the renames into a separate PR please?

Reply:
Sure, I'm a big proponent of focused PR's and I'm not really sure why I let these slip in.  I think I did this early on and never backed them out:)  Anyways, I'll back these out and propose later.